tests/librispeech/Makefile

```
@@ -0,0 +1,22 @@
+TAR_URL = https://www.openslr.org/resources/12/test-clean.tar.gz
+
+all: eval
+
+eval:
+	$(MAKE) -f eval.mk
+
+clean:
+	$(MAKE) -f eval.mk clean
+
+setup-venv:
+	python3 -m venv venv
+	./venv/bin/pip install -r requirements.txt
```

I'm not very experienced with Python so I might be wrong, but isn't it required to source the virtual environment like this:

```suggestion
setup-venv:
	python3 -m venv venv
	source ./venv/bin/activate
	pip install -r requirements.txt
```

---

> but isn't it required to source the virtual environment like this:

Since Makefile executes each line in a separate shell, the third 
line (`pip install`) would be executed outside of the virtualenv.

*Result on Ubuntu 24.04 (with `python3-venv`)*

```console
$ make setup-venv
python3 -m venv venv
source ./venv/bin/activate
pip install -r requirements.txt
bash: line 1: pip: command not found
make: *** [Makefile:16: setup-venv] Error 127
```

Calling `./venv/bin/pip` is sort-of a common workaround to
handle this Makefile issue (this is why I called `./venv/bin/python`
in another part of the code).

> For example, I use pyenv to manage my python environments and this is not compatible with that workflow

I'm going to set up a `pyenv` environment myself, and make sure
that the Python part works both on `pyenv` and `python+virtualenv`.
Wait for me!

---

I see.

Btw, there is also the solution to leave the python environment configuration out of the scripts and just assume that `python` is available and the user has done the necessary steps to install the dependencies.